packages/esm-pharmacy-app/src/pharmacy-metrics/pharmacy-metrics-header.component.tsx

```diff
@@ -0,0 +1,142 @@
+/* eslint-disable no-console */
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { ArrowRight } from '@carbon/react/icons';
+import styles from './pharmacy-header.scss';
+import { Button, TextInput, Modal, Select, ComboBox, SelectItem, DatePickerInput, DatePicker } from '@carbon/react';
+
+const MetricsHeader = () => {
+  const { t } = useTranslation();
+  const metricsTitle = t('pharmacySummary', 'Pharmacy Summary');
+  const [isModalOpen, setIsModalOpen] = useState<boolean>(false);
+  const filterItems = (menu) => {
+    return menu?.item?.toLowerCase().includes(menu?.inputValue?.toLowerCase());
+  };
+
+  return (
+    <div className={styles.metricsContainer}>
+      <span className={styles.metricsTitle}>{metricsTitle}</span>
+      <div className={styles.actionBtn}>
+        <Button
+          kind="tertiary"
+          renderIcon={(props) => <ArrowRight size={16} {...props} />}
+          iconDescription={t('enroll', 'Enroll new body')}
+          // onClick={() => setIsModalOpen(true)}
+        >
+          {t('tagPharmacy', 'Tag pharmacy')}
+        </Button>
+      </div>
+      <Modal
+        primaryButtonText="ADMIT NEW BODY"
+        secondaryButtonText="CANCEL"
+        open={isModalOpen}
+        onRequestClose={() => setIsModalOpen(false)}
+        modalLabel="ADMIT BROUGHT IN BODY"
+        modalHeading="ADMISSION"
+        hasScrollingContent>
+        <TextInput
+          data-modal-primary-focus
+          id="text-input-1"
+          labelText="Given Name*"
+          placeholder="Enter your given name"
+          required
+          style={{
+            marginBottom: '1rem',
+          }}
+        />
+        <TextInput
+          data-modal-primary-focus
+          id="text-input-1"
+          labelText="Middle Name"
+          placeholder="Enter your middle name"
+          required
+          style={{
+            marginBottom: '1rem',
```

Comment:
```suggestion
            marginBottom: '1rem',
```

should we extract this to the css file